lib/model/entity/Profiel.class.php

```diff
@@ -191,7 +192,29 @@ class Profiel extends PersistentEntity implements Agendeerbaar {
 		'novietSoort' => array(T::String),
 		'kgb' => array(T::Text, true)
 	);
-	/**
+
+	public function __get($name) {
```

Comment:
Ik weet niet of ik dit nu zo'n mooie oplossing vind. Probleem is dat code op de stek hierdoor ook niet meer bij de attributen van het profiel kan als het ingelogde lid geen toestemming heeft om deze te zien. 

Voorbeeld: De maalcie fiscus wil de roodschopper gebruiken leden met laag saldo een mailtje te sturen. Het lid heeft zijn emailadres afgeschermd. De roodschopper kan nu ook geen mail sturen naar het lid omdat getPrimaryMail de string 'CENSUUR' teruggeeft.

Misschien is het mooier om een soort wrapper class ZichtbaarProfiel te maken. Aan je view geef je dan straks gewoon iets als new ZichtbaarProfiel(profiel) mee in plaats van profiel.